code/modules/modular_computers/file_system/programs/antagonist/revelation.dm

```diff
@@ -17,6 +17,13 @@
 
 /datum/computer_file/program/revelation/proc/activate()
 	if(computer)
+		if(istype(computer, /obj/item/modular_computer/tablet/integrated)) //If this is a borg's integrated tablet
+			var/obj/item/modular_computer/tablet/integrated/modularInterface = computer
+			to_chat(modularInterface.borgo,"<span class='danger'>sudo rm -rf /</span>")
+			sleep(20)
+			modularInterface.borgo.adjustFireLoss(200)
+			return
```

Comment:
Avoid the `sleep()` proc, use a timer + callback instead.

Reply:
Now uses a timer to call death, rather than dealing wire damage (the damage wasn't the important part anyway).